packages/files/_test.pony

```diff
@@ -189,10 +194,37 @@ class iso _TestSymlink is UnitTest
       target_path.symlink(link_path),
       "could not create symbolic link to: " + target_path.path)
 
+
+class iso _TestFilePathFrom is UnitTest
+  fun name(): String => "files/FilePath.from-success"
+  fun apply(h: TestHelper) =>
+    try
+      let path = "tmp.filepath"
+      let ambient = h.env.root as AmbientAuth
+      let filepath = FilePath(FileAuth(ambient), path)
+      h.assert_no_error({()? => FilePath.from(filepath, path)? })
+    else
+      h.fail("Unhandled error!")
+    end
+
+class iso _TestFilePathFromError is UnitTest
+  fun name(): String => "files/FilePath.from-error"
+  fun apply(h: TestHelper) =>
+    try
+      let path = "tmp.filepath"
+      let ambient = h.env.root as AmbientAuth
+      let filepath = FilePath(FileAuth(ambient), path)
+      h.assert_error({()? => FilePath.from(filepath, "/")? })
```

Comment:
it took me a minute to figure out why this should be an error.  i think a comment on the test class explaining why an error is expected would be good. i think the same but in reverse could be said for the other test.

also as part of that I think these could have better names.

something like,

"test file path from errors when X" 

and

"test file path from doesn't error when Y"

the current names say what the expected outcome is, not what is being tested.